src/github.com/satori/go.uuid/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (C) 2013-2015 by Maxim Bublis <b@codemonkey.ru>
```

Comment:
Whoops, I'll remove this too from this branch.  It is only used by tests anyway.